File: Python Codes/PrimeFactor.py
import math as m
num = int(input("Enter a number: "))
n = 2
while n<=num:
    if num%n==0:
        i = 2
        isPrime = True
        while i<=(int(m.sqrt(n))):
            if n%i==0:
                isPrime = False
                break
            i += 1
        if isPrime:
            print(n)
    n += 1

File: C program/PrimeFactor.cpp
#include<stdio.h>
#include<math.h>
int main(){
    int num;
    printf("Enter a number: ");
    scanf("%d",&num);
    int i,j;
    printf("Prime factors of %d are: \n",num);
    for(i=2;i<=num;i++){
        if(num%i==0){
            for(j=2;j<=sqrt(i);j++){
                if(i%j==0){
                    break;
                }
            }
            if(j>sqrt(i)){
                printf("%d ",i);
            }
        }
    }
    return 0;
}